app/business/repositories/IAuthRepository.scala

```diff
@@ -0,0 +1,29 @@
+package business.repositories
+
+import business.models._
+import common.LastError
+
+import scala.concurrent.Future
+
+/**
+ * Created by liviuignat on 05/04/15.
+ */
+trait IAuthRepository {

```

Comment:
I actually LOVE the [I]Some_Interface prefix in any language. About separation probably not :)